backend/infrahub/graphql/mutations/branch.py

```diff
@@ -76,6 +77,12 @@ async def mutate(
         model = BranchCreateModel(**data)
         await apply_external_context(graphql_context=graphql_context, context_input=context)
 
+        try:
+            await Branch.get_by_name(db=graphql_context.db, name=model.name)
+            raise ValidationError(f"The branch {model.name}, already exist")
```

Comment:
I have a doubt whether even with a ValidationError this error would appear in server logs or not. Can you / did you confirm that they do not show up?

Reply:
Yes it doesn't show up in the logs anymore, it was in the prefect logs before this fix. I am running my server using `infrahub server start --debug`